emergency-protocol/scripts/consumerValidator.ts

```diff
@@ -0,0 +1,69 @@
+import { ethers, starknet, network } from 'hardhat'
```

Comment:
The idea here was to create a scripte that update the status every min (so send a message from L1 to L2 every min) and then the user can run getLatestPrice which will return the round.answer if the status is UP and if the status is Down then it will fail.